src/components/Kv/KvMap.vue

```diff
@@ -86,14 +86,14 @@ export default {
 		 */
 		lat: {
 			type: Number,
-			default: 37.700091
+			default: null
```

Comment:
Having defaults in lat + long was causing them to get set during the component creation and not updated when the props came in. Added the watcher and ability to initialize using those or on mounted.